IotControlCenterUi/src/config.ts

```diff
@@ -2,6 +2,12 @@ const config = {
   LED_STRIP_ENDPOINT: "http://10.0.0.34/lighting/ledstrip",
   BULB_1_ENDPOINT: "http://10.0.0.34/lighting/bulb1",
   BULB_2_ENDPOINT: "http://10.0.0.34/lighting/bulb2",
+  OCEAN_SCENE_ENDPOINT: "http://10.0.0.34/lighting/scene/ocean",
```

Comment:
I would suggest you make this cleaner by making one URL for the scene, then adding the route as needed. IE making a `SCENE_ENDPOINT` then when making posts, `post(data, SCENE_ENDPOINT + "/peachy")`. Would make this file cleaner